.github/workflows/arm64-build-checks.yml

```diff
@@ -0,0 +1,50 @@
+name: arm64 build checks
+
+on:
+  pull_request:
+    branches: [ master ]
```

Comment:
Do we really need to specify the branch? It will be more flexible to have just
```
on: [push, pull_request]
```

Reply:
![image](https://user-images.githubusercontent.com/1736354/89884499-396e6880-dbfc-11ea-8f19-0afddf1ec85f.png)

If we enable the `push` and `pull_request` labels, will triger two same tests on every PRs like above. So, maybe we could only change it to `on: [pull_request]` to enable build check on every PR in all branches.

Reply:
Ok, let's keep pull_request only, yes.